R/dist.R

```diff
@@ -430,7 +430,7 @@ bootstrapped_dist_fit <- function(values,  dist = "lognormal",
                                              future.scheduling = Inf,
                                              future.globals = c("values", "bootstraps", "samples",
                                                                 "bootstrap_samples", "get_single_dist"),
-                                             future.packages = "data.table")
+                                             future.packages = "data.table", future.seed = TRUE)
```

Comment:
This seems unrelated to logging - is it intentional?

Reply:
It is - caught as I tested due to an update to `{future}`.

Reply:
Do we need to update the future dependency definition in DESCRIPTION with a minimum version to support future.seed requirement?

Reply:
no its still works with older version but now throws a warning.

Reply:
brill